Tasks/google-play-release/GooglePlay.ts

```diff
@@ -114,6 +113,12 @@ async function run() {
                 tl.debug(`Uploading APK ${apkFile}`);
                 const apk: googleutil.Apk = await googleutil.addApk(edits, packageName, apkFile);
                 tl.debug(`Uploaded ${apkFile} with the version code ${apk.versionCode}`);
+                if ((shouldPickObbForApk(apkFile, mainApkFile)) && (getObbFromParentDirectroy(apkFile) !== null)) {
```

Comment:
This code path should have an L0 added.  Could you please add a Unit Test for this?

Reply:
Could you please guide me on how I can go about doing this?

Reply:
Sure!  The basics are that you run `gulp test` to run the L0's.

You add the test to `Tests\L0.ts` like [this](https://github.com/microsoft/google-play-vsts-extension/blob/6cb76f6e79c779af0d379f2b021e1551d706344c/Tasks/google-play-release/Tests/L0.ts#L73)
You tell the test what you want the inputs and the API results to be in a file like [this](https://github.com/microsoft/google-play-vsts-extension/blob/master/Tasks/google-play-release/Tests/L0DeobfuscationFileNotFound.ts)

A PR that has the change with the test is [here](https://github.com/microsoft/google-play-vsts-extension/commit/9f6d09f61ab8bf23f868e3c561d425078d573347).

Reply:
Sure!  The basics are that you run `gulp test` to run the L0's.

You add the test to `Tests\L0.ts` like [this](https://github.com/microsoft/google-play-vsts-extension/blob/6cb76f6e79c779af0d379f2b021e1551d706344c/Tasks/google-play-release/Tests/L0.ts#L73)
You tell the test what you want the inputs and the API results to be in a file like [this](https://github.com/microsoft/google-play-vsts-extension/blob/master/Tasks/google-play-release/Tests/L0DeobfuscationFileNotFound.ts)

A PR that has the change with the test is [here](https://github.com/microsoft/google-play-vsts-extension/commit/9f6d09f61ab8bf23f868e3c561d425078d573347).